models/azure_project.rb

```diff
@@ -47,9 +47,9 @@ def resource_group
     @metadata['resource_group']
   end
 
-  def daily_report(date=Date.today-2, slack=true, text=true, rerun=false, verbose=false)
+  def daily_report(date=Date.today-3, slack=true, text=true, rerun=false, verbose=false)
```

Comment:
This is different from the definition in `aws_project`. Which one is correct?

Reply:
Missed this one! Should be `DEFAULT_DATE`